src/PayID/pay-id-client.ts

```diff
@@ -112,16 +68,16 @@ export default class PayIdClient {
           return data.addresses[0].addressDetails
```

Comment:
We'll want to remove `addressForPayId` and instead use `cryptoAddressForPayId` when master gets merged in.